reject-escalated-privileges-yaml/reject-escalated-privileges-yaml.rb

```diff
@@ -18,7 +18,7 @@
 def main(gh)
   pattern = Regexp.union(STRING_LIST)
   yaml_files_in_pr(gh).find_all { |file|
-    hash = YAML.load_file(file)
+    hash = YAML.load_file(file) if FileTest.exists?(file)
```

Comment:
This mean hash will be nil when the file doesn't exist, but on the next line you're recursing into it, regardless.

I think a cleaner approach would be to filter the list of files to only process files which exist, in the first place. Something like:

```
yaml_files_in_pr(gh).find_all { |file| FileTest.exists?(file) }.find_all { ...
```

So you know you're only processing files which exist.

Reply:
Good point @digitalronin . I have re-arranged the condition so the following lines are executed only if file exists. Hope that is ok.